sentry_streams/sentry_streams/adapters/arroyo/reduce.py

```diff
@@ -66,6 +75,64 @@ def accumulator(self, result: Any, value: BaseValue[RoutedValue]) -> RoutedValue
         return routed
 
 
+class MetricsAccumulator(Accumulator[Message[Any], Any]):
```

Comment:
hint: `MetricsReportingAccumulator`, `ObservableAccumulator` ? `MetricsAccumulator` seems an accumulator of metrics.